src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 726px;
```

Comment:
don't give hardcode width in px. Try to calculate in variable with all days and gaps in row

Reply:
ok, let it be so